example/lib/main.dart

```diff
@@ -33,13 +33,41 @@ class _MyAppState extends State<MyApp> {
           title: const Text('Plugin example app'),
         ),
         body: Center(
-            child: MaterialButton(
-          onPressed: () {
-            DoordeckFlutterPlugin.showUnlock();
-          },
-          child: Text('Show unlock'),
-        )),
+          child: Padding(
+            padding: const EdgeInsets.all(16.0),
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                MaterialButton(
+                  onPressed: DoordeckFlutterPlugin.showUnlock,
+                  child: Text('Show unlock'),
+                ),
+                _buildUnlockWithUUID(),
+              ],
+            ),
+          ),
+        ),
       ),
     );
   }
+
+  Widget _buildUnlockWithUUID() {
+    final TextEditingController textEditingController = TextEditingController();
+    return Row(
+      children: [
+        Flexible(
+          child: TextField(
+            controller: textEditingController,
+            decoration: InputDecoration(
+              hintText: "Enter your tile id here (UUID)",
```

Comment:
See! Now I'm not sure if we're talking about deviceId or tileId!

Reply:
both can be used. I'll change the hint

Reply:
How does that work? If it's a tile ID you need to resolve it to a device ID by calling some API and that API can return multiple device IDs...